hadoop-ozone/dist/src/main/compose/testlib.sh

```diff
@@ -93,6 +97,12 @@ stop_docker_env(){
 
 ## @description  Generate robot framework reports based on the saved results.
 generate_report(){
-  #Generate the combined output and return with the right exit code (note: robot = execute test, rebot = generate output)
-  docker run --rm -v "$DIR/..:${OZONE_DIR:-/opt/hadoop}" apache/hadoop-runner rebot -d "$RESULT_DIR_INSIDE" "$RESULT_DIR_INSIDE/robot-*.xml"
+  which rebot > /dev/null 2>&1
+  if [ "$?" == "0" ]; then
```

Comment:
shellcheck:8: note: Check exit code directly with e.g. 'if mycmd;', not indirectly with $?. [SC2181]